config/default.json

```diff
@@ -21,7 +21,10 @@
   },
   "hmackey": "YOU MUST CHANGE ME",
   "host": "127.0.0.1",
-  "logging": "mozlog",
+  "log": {
+    "level": "off",
```

Comment:
This line doesn't turn off our logs, only Rocket logs. That's the behavior we want in production, that's why I put it in there. Should I change it anyways?

Reply:
Whoops, my bad! Ignore the stupid!

Reply:
Not really knowing what sort of information is in Rocket logs, but is there any situation where we might want to be able to configure the application to record this log information?

Reply:
@jrgm Rocket's `Normal` logging was just noise, because our logs record basically the same thing but in mozlog format. We may sometimes want to turn on Rocket `Debug` logging for debugging purposes, like mentioned in #139 .